src/main/java/org/broadinstitute/hellbender/tools/exome/allelefraction/AllelicBiasParameters.java

```diff
@@ -0,0 +1,53 @@
+package org.broadinstitute.hellbender.tools.exome.allelefraction;
+
+/**
+ * Encapsulates the global parameters of the allele fraction model: the mean and variance of the common prior on
+ * allelic biases and the outlier probability.
+ *
+ * @author David Benjamin &lt;davidben@broadinstitute.org&gt;
+ */
+public class AllelicBiasParameters {
```

Comment:
Also, not quite sure about the name of the class---is `outlierProbability` really a "bias parameter"?